python/sglang/srt/cpu_utils.py

```diff
@@ -49,3 +51,16 @@ def reset_param_data_if_needed(param_data, dim, start, length):
 
     param_data.narrow(dim, start, length).zero_()
     return
+
+
+def support_amx():
+    return torch._C._cpu._is_amx_tile_supported()
+
+
+def need_weight_pack(weight_tensors):
```

Comment:
simplify `need_weight_pack` to

```
def prepack_weight_if_needed(weight):
    if not cpu_has_amx_support or weight.device != torch.device("cpu")
         return weight;

    return convert_weight_packed(weight)
```

Reply:
Fixed in https://github.com/mingfeima/sglang/pull/8/commits/533027853f5681629db97387d1d693d653808bbf